tests/unit/components/sl-chart-test.js

```diff
@@ -0,0 +1,363 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+let testOptions = {
+    chartOptions: {
+        chart: {
+            type: 'bar'
+        },
+
+        xAxis: {
+            categories: [ 'Apples', 'Bananas', 'Oranges' ]
+        },
+
+        yAxis: {
+            title: {
+                text: 'Fruit Eaten'
+            }
+        }
+    }
+};
+
+let testSeries = [{ 
+    name: 'Alice', 
+    data: [ 1, 0, 4 ]
+}];
+
+moduleForComponent( 'sl-chart', 'Unit | Component | sl chart', {
+	unit: true
+});
+
+test( 'Default classNames are present', function( assert ) {
+    this.subject({ 
+        options: testOptions, 
+        series: testSeries 
+    });
+
+    assert.ok(
+        this.$().hasClass( 'panel' ),
+        'Default rendered component has class "panel"'
+    );
+
+    assert.ok(
+        this.$().hasClass( 'panel-default' ),
+        'Default rendered component has class "panel-default"'
+    );
+
+    assert.ok(
+        this.$().hasClass( 'sl-chart' ),
+        'Default rendered component has class "sl-chart"'
+    );
+
+    assert.ok(
+        this.$().hasClass( 'sl-panel' ),
+        'Default rendered component has class "sl-panel"'
+    );
+});
+
+test( 'Loading state adds loading class', function( assert ) {
+    let component = this.subject({
+        options: testOptions, 
+        series: testSeries 
+    });
+
+    assert.strictEqual(
+        this.$().hasClass( 'sl-loading' ),
+        false,
+        'Default rendered component does not have "sl-loading" class'
+    );
+
+    Ember.run( () => {
+        component.set( 'isLoading', true );
+    });
+
+    assert.ok(
+        this.$().hasClass( 'sl-loading' ),
+        'Rendered component gains "sl-loading" class'
+    );
+});
+
+test( 'updateData() is called once', function( assert ) {
+    let component = this.subject({
+        options: testOptions, 
+        series: testSeries 
+    });
+
+    let spy = sinon.spy( component, 'updateData');
+
+    this.render();
+
+    assert.equal(
+        spy.calledOnce,
+        true,
+        'updateData() is called once'
+    );
+});
+
+test( '"Options" property needs to be an object', function( assert ) {
```

Comment:
There is a an extra space between the `}  catch(` in each of the blocks below that should be removed.